server/models/Comment.js

```diff
@@ -0,0 +1,20 @@
+module.exports = (sequelize, DataTypes) => {
+  const Comment = sequelize.define(
+    'Comment',
+    {
+      body: DataTypes.STRING
+    },
+    {}
+  );
+  Comment.associate = models => {
+    Comment.belongsTo(models.Recipe, {
+      foreignKey: 'recipeId',
+      onDelete: 'CASCADE'
+    });
+    Comment.belongsTo(models.User, {
+      foreignKey: 'userId',
+      onDelete: 'CASCADE'
```

Comment:
**[question]** What other `onDelete` options exist (and would they be more appropriate here, based on requirements)?

Reply:
There are five different onDelete options: `RESTRICT`, `CASCADE`, `SET DEFAULT`, and `NO ACTION`, and `SET NULL`. By default, onDelete is `SET NULL` for belongsTo, hasOne, and hasMany associations. For belongsToMany association, the default is `CASCADE`. There's also an onUpdate which is set to `CASCADE` at default.

http://docs.sequelizejs.com/manual/associations.html
<img width="1156" alt="Screenshot 2019-03-21 at 1 21 40 AM" src="https://user-images.githubusercontent.com/29962968/54727368-b74d5200-4b77-11e9-887c-b185e00ddf33.png">